### PR TITLE
Feature/kpn session enddate

### PR DESCRIPTION
**Pull request by harrymiddelburgetall**
The GetSessionInfo request functionally differ between REST and SOAP interfaces of JasperWireless
- REST-interface returns information about the current **or most recent data session** for a given device.
- SOAP-interface returns the current session information (IP address and session start time) for one or more devices. If the specified device is **not in session, no information** is returned.

Furthermore the SendSmsResponse of the REST interface returns in contradiction to the documentation a  **_smsMessageId_** insteadof a **_smsMsgId_**

### CODE REVIEW

**Comment by harrymiddelburgetall** [comment]
Nightly: https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/535/

File: osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/rest/client/JasperWirelessTerminalRestClient.java [hunk 1]
```diff
@@ -60,11 +60,34 @@ public GetSessionInfoResponse getSession(final String iccId) throws OsgpJasperEx
 
       getSessionInfoResponse = getSessionInfoResponseEntity.getBody();
 
+      getSessionInfoResponse = this.checkOnSessionValidity(getSessionInfoResponse);
+
```
Comment (kroesctrl) [inline osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/rest/client/JasperWirelessTerminalRestClient.java:62]: Now the same variable is used twice, would it not be nice to make a method:
sessionInfoResponse = getValidSessionInfoResponse(getSessionInfoResponseEntity)

**Review by kroesctrl — CHANGES_REQUESTED** [review]

File: osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/rest/client/JasperWirelessTerminalRestClient.java (hunk 1 above)
Comment (harrymiddelburgetall) [reply osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/rest/client/JasperWirelessTerminalRestClient.java:62]: I agree

**Comment by harrymiddelburgetall** [comment]
Nightly: https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/536/

**Review by kroesctrl — APPROVED** [review]

**Review by kroesctrl — APPROVED** [review]

**Pull request merged**